src/components/movieItem.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+
+
+export default class MovieItem extends Component {
+
+    render() {
+        const { movieData } = this.props;
+
+        return (
+            <div className="movieitem-wrapper">
+                <span>Title: { movieData.Title }</span>
```

Comment:
Sorry I could have described it better in our planner. 
I was thinking listing the results in table - which is why I said about columns. Something like this.

Title | Year
------------ | -------------
Best Movie Ever Title | 1972
Sequel of Best Movie | 2005